docs/js-api.md

```diff
@@ -159,7 +159,7 @@ One of `"mouseOver"`, `"rangeSelect"`, `"click"`, `"rawData"`, `"onNewTrack"`, `
       id: string
   }
   ```
-  - For `"rawData"`, the `eventData` stores columnar data that has been used internally in Gosling to display the view:
+  - For `"rawData"` and `"specTraversed"`, the `eventData` stores columnar data that has been used internally in Gosling to display the view:
```

Comment:
I think a separate sentence or a paragraph to describe the new API will be helpful for users. To clarify, `rawData` gives users the data that is shown in the visualization and not the user specification.